brain/compartmentReportReader.h

```diff
@@ -13,6 +13,7 @@
  * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
  * details.
  *
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Comment:
Split this file in one file per class. You may keep CompartmentReportMetadata with CompartmentReport.